polaris-react/src/components/PositionedOverlay/PositionedOverlay.tsx

```diff
@@ -17,12 +17,15 @@ import {
 import type {PreferredPosition, PreferredAlignment} from './utilities/math';
 import styles from './PositionedOverlay.module.css';
 
-type Positioning = 'above' | 'below' | 'cover';
+type Positioning = 'above' | 'below' | 'left' | 'right' | 'cover';
```

Comment:
This isn't a hard opinion, but I think `left` and `right` make sense here because it is _relative_ to the activator 🤔 but I'm not married to that and happy to change it to be consistent with the rest of our positioning props